client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -62,7 +63,9 @@ export class CommentAuthor extends Component {
 			<div className="comment__author">
 				<div className="comment__author-avatar">
 					{ /* A comment can be of type 'comment', 'pingback' or 'trackback'. */ }
-					{ 'comment' === commentType && <Gravatar user={ gravatarUser } /> }
+					{ 'comment' === commentType && !! authorAvatarUrl && <Gravatar user={ gravatarUser } /> }
```

Comment:
Is this a bugfix that slip in?

Reply:
nevermind. went back to https://github.com/Automattic/wp-calypso/pull/19225 for context :)

Reply:
Ah it is sort of a little fix.
Basically if you pass an empty object to `Gravatar`, as it happens when comments load, you obtain a placeholder gravatar (which is the white user icon on a gray background).
It is by itself the appropriate behaviour.

Though, it always bugged me that the gravatar gray is slightly different than Calypso gray, and it was especially obvious to me when the comment is in placeholder state, since it's near the gray placeholder boxes.
By adding that check, we make sure that we only request the gravatar when the comment is actually loaded, but try to use a generic Calypso-gray circle while it loads.